### ISSUE TITLE
rebuild-index has no output

### ISSUE THREAD

**Issue by Thor77** [restic/restic#786]
## Output of `restic version`
```
restic 0.4.0
compiled with go1.7.5 on linux/amd64
```

## Expected behavior
Some helpful output (like progress, status, etc.)

## Actual behavior
No output at all

## Steps to reproduce the behavior
1. run `restic rebuild-index ` on your repo

**Comment by fd0** [comment]
Yeah, that's something we can definitely improve!

**Comment by middelink** [comment]
I think this is an obsolete bug, fixed by #837 

```console
$ restic -r rest:http://qnap.**.********.**:8000 rebuild-index
counting files in repo
[1:07] 100.00%  9022 / 9022 packs
listing old index files
saved new index as 4e07c35b
remove 36 old index files
```

And... (i checked) it is completely silent under -q

**Comment by fd0** [comment]
Ok, cool! I'm going to close this issue :)
